tests/rptest/services/redpanda.py

```diff
@@ -937,6 +945,28 @@ def write_tls_certs(self):
                 os.path.dirname(RedpandaService.TLS_CA_CRT_FILE))
             node.account.copy_to(ca.crt, RedpandaService.TLS_CA_CRT_FILE)
 
+            if self._pandaproxy.pp_client_key is not None:
```

Comment:
I'd break out these changes with the other changes in this commit to add new tests